filament/include/filament/View.h

```diff
@@ -139,6 +139,24 @@ class UTILS_PUBLIC View : public FilamentAPI {
         QualityLevel hdrColorBuffer = QualityLevel::HIGH; //!< quality of the color buffer
     };
 
+    /**
+     * Options for Screen Space Ambient Occlusion (SSAO)
+     * @see setSSAO()
+     */
+    struct SSAOOptions {
+        float radius = 0.3f;    //!< SSAO radius in meters, between 0 and ~5.
```

Comment:
Seems like `radius` is actually getting clamped to 10 in `setSSAOOptions`.

Reply:
good catch.